core/types/src/attributes.rs

```diff
@@ -23,33 +24,31 @@ impl Attributes {
         self
     }
 
-    /// Set the `transform` for the attributes
+    /// Set the extended features request mask (xfrm)
     ///
     /// # Arguments
     ///
-    /// * `transmform` - The transform to be set to the `xfrm` in the attributes
-    pub fn set_transform(mut self, transform: u64) -> Self {
-        self.0.xfrm = transform;
+    /// * `features_mask` - The mask to be set to the `xfrm` in the attributes
+    pub fn set_extended_features_mask(mut self, features_mask: u64) -> Self {
+        self.0.xfrm = features_mask;
         self
     }
 }
 
+/// Miscellaneous select bits for target enclave. Reserved for future extension.
 #[repr(transparent)]
 #[derive(Debug, Clone, Hash, PartialEq, Eq, Default)]
 pub struct MiscellaneousSelect(sgx_misc_select_t);
+
 new_type_accessors_impls! {
     MiscellaneousSelect, sgx_misc_select_t;
 }
 
-impl MiscellaneousSelect {
-    pub fn new(select: u32) -> Self {
-        Self(select)
-    }
-}
-
```

Comment:
Removed the `new` here, and elsewhere, as it was more than one way to do something with the existence of the `From` trait implementation.  Figured the `From` trait provides more utility